.github/config.yml

```diff
@@ -4,7 +4,7 @@
 
 # Comment to be posted to on first time issues
 newIssueWelcomeComment: >
-  Welcome! 👋 Thanks for opening your first issue here! And to ensure the community is able to respond to your issue, be sure to follow the issue template if you haven't already.
+  Welcome! 👋 Thanks for opening your first issue here! And to ensure the community is able to respond to your issue, be sure to be sure to fill out the issue form template inputs accurately if you haven't already.
```

Comment:
`be sure to` is duplicated here and I think the phrasing can be simplified a bit.

`accurately` feels a bit strong here and I'm not that concerned if something was not done perfectly when someone is filing an issue.

`if you haven't already` feels unneeded here as with the form inputs the user will have already used the template so I think this can be omitted.